spring/spring4/spring4-remote/src/test/java/org/infinispan/spring/provider/sample/SampleHotrodServerLifecycleBean.java

```diff
@@ -46,8 +47,8 @@ public void afterPropertiesSet() throws Exception {
       cacheManager.defineConfiguration(remoteBackupCacheName, HotRodTestingUtil.hotRodCacheConfiguration().build());
       cacheManager.defineConfiguration(customCacheName, HotRodTestingUtil.hotRodCacheConfiguration().build());
       HotRodServerConfigurationBuilder hcb = new HotRodServerConfigurationBuilder();
-      hcb.port(15233);
-      hotrodServer = HotRodClientTestingUtil.startHotRodServer(cacheManager, hcb);
+      hcb.port(PORT);
+      hotrodServer = HotRodClientTestingUtil.startHotRodServer(cacheManager, PORT, hcb);
```

Comment:
Nevermind, I was about to suggest not to hardcode the port in the test, but rely in the free port discovered, but I now see this test uses a properties file